core/src/main/scala/weaponregex/model/mutation/TokenMutator.scala

```diff
@@ -18,6 +19,18 @@ trait TokenMutator {
     */
   def description: String
 
+  /** Provide a description for a specific mutation based on a given token and location. By default, it is the same as
+    * the description of the mutator with the addition of the start position from the given location.
+    *
+    * @param token
+    *   The token to be mutated
+    * @param location
+    *   The location of the mutation
+    * @return
+    *   A description of the mutation
+    */
+  def describeMutation(@nowarn token: RegexTree, location: Location): String = s"$description at ${location.start}"
```

Comment:
I'm not too sure about this pattern. It seems like it's doing almost the same thing as the `description` method. It also seems inconsistent in adding the location at the end for some mutations, and a different description for some others.

Reply:
I’m also still puzzled about the description pattern consistency. I’m thinking of always having the location first, but it reads better like this, in my opinion. Any suggestions?
```scala
"Remove the child Character at [0:1, 0:2) from the character class"
"[0:1, 0:2) Remove a child Character from the character class"
```